products/brms/get-started.adoc

```diff
@@ -8,7 +8,7 @@ The purpose of this kit is to provide the means to get started with the http://d
 
 It will present the available content in a logical order, providing a learning path for first time users. With the video tutorials designed to be non-specific to a version, these initial steps will save you time spinning up your first process application or project. Just follow the numbered steps below and you will be on your way to mastering JBoss BRMS.
 
-This starters kit is currently targeting the product https://developers.redhat.com/download-manager/file/jboss-brms-6.1.0.GA-installer.jar[JBoss BRMS 6.1].
+This starters kit is currently targeting the product #{site.download_manager_file_base_url}/jboss-brms-6.1.0.GA-installer.jar[JBoss BRMS 6.1].
```

Comment:
You need to prefix with 'link:'. See that it's not rendering here: http://it-developers-pr.stage.redhat.com/pr/674/build/385/products/brms/get-started/. This should work:

      link:#{site.download_manager_file_base_url}/jboss-brms-6.1.0.GA-installer.jar[JBoss BRMS 6.1].